guides.xml

```diff
@@ -15,4 +15,70 @@
     <output-format>html</output-format>
     <output-format>singlepage</output-format>
     <output-format>interlink</output-format>
+    <inventory id="t3coreapi"
```

Comment:
Please sort them alphabetically by id. Also add some breaking and comments between different types of manuals, like

- Manuals
- System extension
- Other

This improves readability a lot.